cart/cart.html

```diff
@@ -22,10 +22,10 @@
                     <small class="sm">Let's shop</small></button>
 
             </div>
-            <section>
-                <input type="text" class="input-text md" placeholder="Search...">
-                <a href="/login/login.html"><button class="primary btn md login without-shadow ">Login</button></a>
-            </section>
+            <input type="text" class="input-text md" placeholder="Search...">
+            <a href="/logout/logout.html"><button class="material-icons primary btn size-12 login without-shadow ">
+                    logout
+            </button></a>
             <div class="nav-section-right ">
```

Comment:
Formatting could be done better

Reply:
Will do that. Thanks for the suggestion.